README.md

```diff
@@ -1,7 +1,11 @@
 # TypedStruct
 
 [![Build Status](https://travis-ci.com/ejpcmac/typed_struct.svg?branch=develop)](https://travis-ci.com/ejpcmac/typed_struct)
-[![hex.pm version](http://img.shields.io/hexpm/v/typed_struct.svg?style=flat)](https://hex.pm/packages/typed_struct)
+[![Module Version](https://img.shields.io/hexpm/v/typed_struct.svg)](https://hex.pm/packages/typed_struct)
+[![Hex Docs](https://img.shields.io/badge/hex-docs-lightgreen.svg)](https://hexdocs.pm/typed_struct/)
+[![Total Download](https://img.shields.io/hexpm/dt/typed_struct.svg)](https://hex.pm/packages/typed_struct)
+[![License](https://img.shields.io/hexpm/l/typed_struct.svg)](https://github.com/ejpcmac/typed_struct/blob/master/LICENSE)
+[![Last Updated](https://img.shields.io/github/last-commit/ejpcmac/typed_struct.svg)](https://github.com/ejpcmac/typed_struct/commits/master)
```

Comment:
Maybe point `develop` instead, as project development occurs there. Also, while such badge is interesting for evaluating the maintenance status of big projects, I doubt it has a big interest for a library like TypedStruct, which should be mostly stable and then not updated that often.

Other badges are pretty interesting otherwise, I will adopt them too on other projects. Why “Module Version” instead of “Hex.pm version”?

Reply:
> Maybe point `develop` instead, as project development occurs there. Also, while such badge is interesting for evaluating the maintenance status of big projects, I doubt it has a big interest for a library like TypedStruct, which should be mostly stable and then not updated that often.

Done.

> Other badges are pretty interesting otherwise, I will adopt them too on other projects. Why “Module Version” instead of “Hex.pm version”?

Just my typical naming convention. I've renamed it to `Hex.pm version` instead.